plenum/test/view_change/test_no_view_change_while_catchup.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from plenum.common.startable import Mode
+from plenum.test.helper import checkViewNoForNodes, waitForViewChange
+from plenum.test.test_node import ensureElectionsDone
+from plenum.test.view_change.helper import do_view_change, revert_do_view_change
+from plenum.test.waits import expectedPoolViewChangeStartedTimeout
+
+
+@pytest.fixture(params=['starting', 'discovering', 'discovered', 'syncing'])
```

Comment:
Do we really need strings here and lot's of if/else? Would the following do the job?
```
@pytest.fixture(params=[Mode.starting, Mode.discovering...])
def mode(request)
    return request.param
```

Reply:
Done